### ISSUE TITLE
DAC 12 - Medium Priority WCAG Level AA - Non-Descriptive Names

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#12]
<img width="285" alt="DAC_Non_Descriptive_Names_01" src="https://user-images.githubusercontent.com/75850106/155557360-03a111e2-e8cf-4658-b0b6-36322030b985.png">

### Issue
The name of the select element that is used to choose the place a user wants to see how it compares for income deprivation measures, may not have a descriptive name for screen reader users. This is because there is another element that serves a different purpose and has the same name.

<img width="601" alt="DAC 12-1 Current code ref" src="https://user-images.githubusercontent.com/75850106/155557301-03f69705-385d-4878-963e-da4a8bb776f2.png">

### Suggested Solution
Interactive elements that serve a different purpose should have unique accessible names. We recommend that the visible name ‘Select an area on the map or drop-down menu to see how it compares for income deprivation measures.’ is used to name the select element instead.

<img width="599" alt="DAC 12-2 Current code ref" src="https://user-images.githubusercontent.com/75850106/155557325-b640a6a3-b7be-4a4b-9502-280c21510c94.png">
